ucm2/NXP/iMX8/Librem_5/HiFi.conf

```diff
@@ -0,0 +1,180 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Digital Playback Volume' 100,100"
+		cset "name='MIXINL IN2L Switch' off"
+		cset "name='MIXINL IN3L Switch' off"
+		cset "name='MIXINR IN2R Switch' off"
+		cset "name='MIXINR IN3R Switch' off"
+		cset "name='INPGAR IN1R Switch' off"
+		cset "name='INPGAR IN2R Switch' off"
+		cset "name='INPGAR IN3R Switch' on"
+		cset "name='INPGAR IN4R Switch' off"
+		cset "name='INPGAL IN1L Switch' off"
+		cset "name='INPGAL IN2L Switch' off"
+		cset "name='INPGAL IN3L Switch' off"
+		cset "name='INPGAL IN4L Switch' off"
+		cset "name='Input Mixer Switch' off,on"
+		cset "name='SPKOUTL PGA' Mixer"
+		cset "name='SPKOUTR PGA' Mixer"
+	]
+
+	DisableSequence [
+		cset "name='Digital Playback Volume' 0,0"
```

Comment:
The volumes should not be touched in enable/disable sequences. Keep the user settings.